runway/hooks/staticsite/auth_at_edge/templates/parse_auth/__init__.py

```diff
@@ -0,0 +1,138 @@
+# pylint: skip-file
+"""Parse the given Cognito/Local authorization information.
+
+Ensure the parsed Cognito and Local authorization information from the retrieved
+query string parameter and cookie headers matches our expectations. If it doesn't
+then inform the user of a bad request, otherwise retrieve the Cognito tokens to
+add to the cookie headers.
+
+NOTE: pylint: skip-file has been added to this file to avoid an issue with a false
```

Comment:
This looks old to me now - should be possible to drop this now

Reply:
Still needed. It picks up on the redirect return values. Large discussion with Kyle about this in an earlier conversation. Leaving in-tact.